src/test/ui/associated-types/associated-types-project-from-hrtb-in-fn-body.nll.stderr

```diff
@@ -1,24 +1,24 @@
 error: unsatisfied lifetime constraints
-  --> $DIR/associated-types-project-from-hrtb-in-fn-body.rs:32:29
+  --> $DIR/associated-types-project-from-hrtb-in-fn-body.rs:32:19
    |
 LL | fn bar<'a, 'b, I : for<'x> Foo<&'x isize>>(
    |        --  -- lifetime `'b` defined here
    |        |
    |        lifetime `'a` defined here
 ...
 LL |     let z: I::A = if cond { x } else { y };
-   |                             ^ assignment requires that `'a` must outlive `'b`
+   |                   ^^^^^^^^^^^^^^^^^^^^^^^^ requires that `'a` must outlive `'b`
```

Comment:
🎉

Reply:
oops apparently that :tada: was premature. :smile:  Luckily this change was not the point per se of the PR; I just (subjectively) thought it might have been a happy accident.